Core/functions.php

```diff
@@ -193,7 +189,7 @@ function getOptionFromInteraction(Collection|Interaction $options, string ...$na
             $options = $option?->options;
         }
 
-        if ($options === null || $option === null) {
```

Comment:
wouldn't comparing to null be faster cause there's no function call overhead, no?

Reply:
Nanoseconds faster, probably.

If you want to optimize this, then there should be two `if`s with `break` in them. The first one should happen right after `$option` assignment within the loop that checks `$option === null` and `break`s the loop if true, while the other one should be after `$key !== count($names) - 1` check; it should check `if ($options === null)` and then `break`, if I understand the initial intent correctly.

Reply:
Also `count($names) - 1` should be assigned to a variable before the `foreach ()` loop happens for an additional optimization.